### PR TITLE
feat(fedimint-cli): added support to pay a lnurl

### PR DESCRIPTION
**Pull request by douglaz**
Added support to paying lnurls. Very common nowadays.

### CODE REVIEW

File: fedimint-cli/src/client.rs [hunk 1]
```diff
@@ -489,6 +497,51 @@ pub async fn handle_command(
...
 
+async fn get_invoice(info: &str, amount: Option<Amount>) -> anyhow::Result<Bolt11Invoice> {
+    let info = info.trim();
+    match lightning_invoice::Bolt11Invoice::from_str(info) {
+        Ok(invoice) => {
+            debug!("Parsed parameter as bolt11 invoice: {invoice}");
+            match (invoice.amount_milli_satoshis(), amount) {
+                (Some(_), Some(_)) => {
+                    bail!("Amount specified in both invoice and command line")
+                }
```
Comment (benthecarman) [inline fedimint-cli/src/client.rs:508]: if these are equal it should be fine

Comment (elsirion) [reply fedimint-cli/src/client.rs:508]: If the invoice doesn't have an amount the recipient won't enforce it, so we'd trust the gateway to actually send the whole amount. One day the client might do the routing, then this wouldn't be a problem anymore.

Comment (elsirion) [reply fedimint-cli/src/client.rs:508]: Ah, sorry, totally missed that, good point!

**Review by elsirion — APPROVED** [review]

**Comment by fedimint-backports** [comment]
Successfully created backport PR for `releases/v0.2`:
- #3854

**Pull request merged**